docs/src/pages/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Introduction
+weight: -100
+---
+
+The [HealthCare.gov](https://www.healthcare.gov/) design system is a set of open-source design and front-end development resources for creating usable, accessible (e.g., WCAG/Section 508 compliant), responsive, and consistent websites and web-based solutions.
+
+The Healthcare.gov design system is built on the CMS Design System and extends it to support additional CSS and React components.
+
+## Getting started
+
+### Developers
+
+Developers and engineers are encouraged to use React which allows for more complex, interactive components, and many of our React components have built-in accessibility optimizations.
```

Comment:
Hm, I feel like this info is duplicated from the [`Using components` page](https://design.cms.gov/startup/components/). Maybe we can just keep it simple and just have two bullet points like on [this page](https://design.cms.gov/startup/installation/)?

Reply:
### Developers

* **Recommended** [Install the `@cmsgov/design-system` NPM package](https://design.cms.gov/startup/installation/#install-using-npm)

* [Download the design system zip](https://design.cms.gov/startup/installation/#download-zip)
```

Reply:
I think from Kristen's perspective she was interested in having a bit more of the why on this page 
See the comments in the confluence page here https://confluence.cms.gov/display/HCDSG/Introduction+Page 

I'm ok with simplifying though

Reply:
Ok, I think this is fine we can always iterate more in the future. Based on the discussion from the confluence theres a lot more to figure out in general, the audience, what info we want, etc.

Reply:
One note though, the download the design system zip link should definitely go to our install page, this would address [Melissa's feedback here](https://confluence.cms.gov/display/HCDSG/Introduction+Page?focusedCommentId=382049021#comment-382049021). Right now we just have a direct download link and no instructions

Reply:
Thanks @bernardwang updated the NPM and download links to go to those sections of the installation page 👍